lesson2.2/webpack.config.js

```diff
@@ -0,0 +1,35 @@
+const { resolve } = require('path'); 
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer')
+
+module.exports = {
+    entry: "./js/switch.js",
+    output: {
+        path: resolve(__dirname, 'build'), 
+        filename: "switch.[contenthash].js"  
+    },
+    module: {
+        rules: [ 
+            {
+                test: /\.(png|jpe?g|gif)$/i,
```

Comment:
А почему mp3 нету в сборке?
```suggestion
                test: /\.(png|jpe?g|gif|mp3)$/i,
```